packages/cli/src/prompts/prompt.ts

```diff
@@ -114,20 +115,22 @@ export function proxiesList(pickProxyBy: string, network: string, packageFile?:
 export function contractsList(name: string, message: string, type: string, source?: string): { [key: string]: any } {
   const localPackageFile = new ZosPackageFile();
   const contractsFromBuild = Truffle.getContractNames();
+  const contractsFromLocal = Object
+    .keys(localPackageFile.contracts)
+    .map(alias => ({ name: localPackageFile.contracts[alias], alias }))
+    .map(({ name: contractName, alias }) => {
+      const label = contractName === alias ? alias : `${alias}[${contractName}]`;
+      return { name: label, value: alias };
+    });
 
   // get contracts from `build/contracts`
-  if (!source || source === 'fromBuildDir') {
+  if (!source || source === 'built') {
```

Comment:
Is this option used at all? If not, can we remove it?

Reply:
Mmh, I believe we are not using it! My bad!